src/CRUDlex/ServiceProvider.php

```diff
@@ -306,6 +307,24 @@ public function getEntities() {
     }
 
     /**
+     * Getter for the entitis fot the navigation bar.
+     *
+     * @return string[]
+     * a list of all available entity names with theirs group
+     */
+    public function getEntitiesNavBar() {
```

Comment:
This method should get an explicit test: EntityDefinitionTest::testGetEntitiesNavBar